satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -619,4 +619,26 @@ update offer (
 	where offer.expires_at >= offer.created_at
 )
 
-delete offer ( where offer.id = ? )
\ No newline at end of file
+delete offer ( where offer.id = ? )
+
+//--- user credit table ---//
+
+model user_credit (
+	key		user_id
+
+	field	user_id user.id	cascade
+	field	offer_id offer.id	cascade
+	field	referred_by user.id	cascade
+	field	credits_earned	int
+	field	credits_used	int
+	field	credits_type	text
+	field	expires_at		timestamp	(autoinsert)
+	field	created_at		timestamp	(autoinsert)
+)
+
+read all ( select user_credit )
```

Comment:
Do we have a use case for getting all the records from this table?

Reply:
Not that I'm aware of. I was following convention when I added this iirc.